autoendpoint/src/extractors/subscription.rs

```diff
@@ -94,24 +94,21 @@ impl FromRequest for Subscription {
             // let's just capture if we have a valid VAPID, as well as what sort of bad sub
             // values we get.
             if let Some(header) = &vapid {
-                let sub = header
-                    .vapid
-                    .insecure_sub()
-                    .map_err(|e: VapidError| {
-                        // Capture the type of error and add it to metrics.
-                        let mut tags = Tags::default();
-                        tags.tags
-                            .insert("error".to_owned(), e.as_metric().to_owned());
-                        app_state
-                            .metrics
-                            .incr_with_tags("notification.auth.error")
-                            .with_tag("error", e.as_metric())
-                            .send();
-                    })
-                    .unwrap_or_default();
+                if let Some(sub) = header.vapid.insecure_sub().inspect_err(|e: &VapidError| {
+                    // Capture the type of error and add it to metrics.
+                    let mut tags = Tags::default();
+                    tags.tags
+                        .insert("error".to_owned(), e.as_metric().to_owned());
+                    app_state
+                        .metrics
+                        .incr_with_tags("notification.auth.error")
+                        .with_tag("error", e.as_metric())
+                        .send();
+                })? {
```

Comment:
This was eating the error previously w/ `unwrap_or_default`

Reply:
Ah, right. I was thinking we wanted it to be raised, but not here. We actually want to eat the error after we add the metric. (sigh)